service/deleteTrackPlay/service.ts

```diff
@@ -1,11 +1,11 @@
+import database from "../../database/database";
 import { NotFoundError } from "../../errors/NotFoundError";
-import { TrackPlay } from "../../models/TrackPlay";
 
 export const deleteTrackPlayService = async (id: string) => {
-  const trackPlayExists = await TrackPlay.exists({ _id: id });
+  const trackPlayExists = await database.existsTrackPlay(id);
   if (!trackPlayExists) {
     throw new NotFoundError();
   }
```

Comment:
Maybe checking if track play exists before deleting is not necessary, maybe it's possible to get this information while trying to delete the track play (it's something to verify in the future)